interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a code management software. Git is software and Github is the technology that runs git. 
```

Comment:
I would really like to see you interview answers expanded on.  Think of this as if you are in an interview and are asked these questions, how would you respond?  Expanding on this will show your interviewer that you are excited and passionate about coding and display your knowledge.